src/index.js

```diff
@@ -15,3 +16,5 @@ module.exports = async function ({ github, context, inputs, packageVersion }) {
 
   logError('Unsupported event')
 }
+
+module.exports.build = build
```

Comment:
ok this is not super, in the sense that I wonder if along this change we shall consider also splitting the other two in two functions instead of having the multiplexing logic in the main export. but ok, we can keep it as it is for now

Reply:
I removed the multiplexing logic and changed the other exports. It was a pretty simple change so I think it won't cause any issues but I can just revert it if you think it's a bad idea